java/BinExport/src/main/java/com/google/security/binexport/BinExport2Builder.java

```diff
@@ -347,7 +353,12 @@ private void buildCallGraph() throws CancelledException {
       }
       // TODO(cblichmann): Check for artificial names
       // Mangled name always needs to be set.
-      vertex.setMangledName(func.getName());
+      if (func.getParentNamespace() != null && !func.getParentNamespace().getName().equals("Global") 
```

Comment:
To be extra defensive, please use `"Global".equals(parentNamespace.getName())`

Reply:
Resolved in latest commit.